tests/xml/data.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0"?>
+<data xmlns="https://schemas.errlock.org/xml/data"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="https://schemas.errlock.org/xml/data https://schemas.errlock.org/xml/data.xsd">
+	<meta>
+		<title>Issue</title>
+		<description>
+			This resource represents a single issue with its data and related
+			actions.
+		</description>
+		<controls>
```

Comment:
Could we use [XLink](https://www.w3.org/TR/xlink11/) for that?

Reply:
Well, we'll end up using only xlink:type="simple" anyway, so forget that.

Maybe we can find a use for extended links, (navigating through the api?) but let's see about that later.